app/src/main/java/com/vitorpamplona/amethyst/ui/note/ChatroomCompose.kt

```diff
@@ -82,7 +82,7 @@ fun ChatroomCompose(baseNote: Note, accountViewModel: AccountViewModel, navContr
         channel?.let { channel ->
             var hasNewMessages by remember { mutableStateOf<Boolean>(false) }
 
-            LaunchedEffect(key1 = notificationCache) {
+            LaunchedEffect(key1 = notificationCache, key2 = note) {
```

Comment:
There was a pre-existing bug I noticed when coding this PR, that is, when a new message comes in while you are looking at the messages tab, the notification bubble does not re-appear unless you leave and come back, either by going to the other tab (Known/New), or going to another screen, etc.

If we key off of the note directly, the recomposition causes `hasNewMessages` to be re-evaluated with the new note so that the bubble can come back straight away.

Reply:
Nice find!